File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.kt
class Solution {
    fun solution(s: String): Boolean {

    val size = s.split("").filter { it == "0" || it == "1" || it == "2" || it == "3" || it == "4" || it == "5" || it == "6" || it == "7" || it == "8" || it == "9"}.size
    
    return s.length == size && (s.length == 4 || s.length == 6)
}
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.swift
func solution(_ s:String) -> Bool {
    let firstCondition = s.count == 4 || s.count == 6
    let secondCondition = s.map { String($0) }.filter { Int(UnicodeScalar($0)!.value) < 48 || Int(UnicodeScalar($0)!.value) > 57 }.count == 0
    
    return firstCondition && secondCondition
}